ls1intum/Artemis
# `Communication`: Remove context duplication while editing answers

### Checklist
#### General



#### Client

### Motivation and Context
While editing replies, the context is shown twice; both in the answer's body and the markdown input. This causes information redundancy.

### Description
- We hide the answer body when the markdown is displayed while editing answers.
- A small bugfix is introduced that prevents the posting edit/delete icons found in the header from moving when the posting creation date tooltip is displayed.

### Steps for Testing

Prerequisites:
- 1 Student
-  1 Course with communication enabled

1. Log in to Artemis
2. Navigate to the Course communication page
3. Edit an answer
4. Make sure that the answer body is not displayed while editing the answer
5. Make sure that the answer body is displayed after editing is completed or canceled.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage
Not changed

### Screenshots

### Old
![image](https://user-images.githubusercontent.com/67344158/184930629-a46166ad-b4ee-45f0-a96c-9d513529cb85.png)

### New
![image](https://user-images.githubusercontent.com/67344158/184939950-81999605-c7d0-47ff-bdab-c350c962bccd.png)
